ProjectSource/src/test/java/com/kayako/sdk/ParserFactoryTest.java

```diff
@@ -0,0 +1,222 @@
+package com.kayako.sdk;
+
+import com.kayako.sdk.base.parser.DataNodeParser;
+import com.kayako.sdk.error.response.*;
+import com.kayako.sdk.helpcenter.articles.ArticleParser;
+import com.kayako.sdk.helpcenter.category.CategoryParser;
+import com.kayako.sdk.helpcenter.locale.LocaleParser;
+import com.kayako.sdk.helpcenter.search.SearchArticleParser;
+import com.kayako.sdk.helpcenter.section.SectionParser;
+import com.kayako.sdk.helpcenter.user.UserMinimalParser;
+import com.kayako.sdk.messenger.attachment.AttachmentParser;
+import com.kayako.sdk.messenger.attachment.ThumbnailParser;
+import com.kayako.sdk.messenger.conversation.ConversationParser;
+import com.kayako.sdk.messenger.conversation.fields.priority.PriorityParser;
+import com.kayako.sdk.messenger.conversation.fields.readmarker.ReadMarkerParser;
+import com.kayako.sdk.messenger.conversation.fields.status.StatusParser;
+import com.kayako.sdk.messenger.conversation.fields.team.TeamParser;
+import com.kayako.sdk.messenger.conversationstarter.ConversationStarterParser;
+import com.kayako.sdk.messenger.message.MessageParser;
+import static org.hamcrest.CoreMatchers.instanceOf;
+
+import com.kayako.sdk.messenger.rating.RatingParser;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Locale;
+
+public class ParserFactoryTest {
+
+    private Locale locale;
+
+    @Before
+    public void setup() {
+        locale = Locale.ENGLISH;
+    }
+
+    @Test
+    public void getErrorListParser() {
```

Comment:
✅ Good to see tests are being added, therefore, increasing the overall quality of the project.

**No change suggested, looks good!**	  

<a id="aa883c60-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />